### ISSUE TITLE
Second browser not opening

### ISSUE THREAD

**Issue by lhrolim** [Microsoft/cordova-simulate#255]
I´m running a "simulate android command" on my cordova project 

It was created a long time ago using TACO on vs2015, but which now runs fine on VS2017 community.

Only one chrome instance is openend and as a consequence the cordova events are not triggered.  

I´ve tried to delete node_modules and reinstalling the global module, with no luck.

I´m running on top of nodejs 8.9.1

Any hints on how to fix that?

**Comment by jwargo** [comment]
I'll see if I can reproduce with an old project.

**Comment by ezze** [comment]
I'm new to `cordova-simulate` and tried to run some Cordova project with it right now. Seems that I have a similar issue. Here is my output:

```
$ ./node_modules/.bin/simulate --target=chromium
SIM: Server started:
- App running at: http://localhost:8000/index.html
- Sim host running at: http://localhost:8000/simulator/index.html
SIM: Preparing platform 'browser'.
SIM: Preparing platform 'browser'.
SIM WARNING: Preparing platform 'browser' failed: Command failed: cordova prepare browser
SIM: Creating app-host.js: Existing file found and is up-to-date.
SIM: Injecting app-host into /home/ezze/Development/nokianproject-android/platforms/browser/www/index.html
200 /index.html (gzip)
304 /css/themes/nokian.min.css
304 /css/themes/jquery.mobile.icons.min.css
304 /css/jquery.mobile.structure-1.4.5.min.css
304 /css/index.css
304 /simulator/thirdparty/socket.io.js
304 /cordova.js
304 /js/jquery-2.1.1.js
304 /js/jquery.mobile-1.4.4.min.js
304 /js/page.js
304 /js/request.js
304 /js/synch.js
304 /js/authentification.js
304 /img/logo.jpg
200 /simulator/app-host.js (gzip)
304 /css/themes/images/ajax-loader.gif
SIM: APP_HOST connected to the server
304 /cordova_plugins.js
304 /plugins/cordova-fabric-plugin/www/FabricPlugin.js
304 /plugins/cordova-fabric-plugin/www/FabricPlugin.Answers.js
304 /plugins/cordova-fabric-plugin/www/FabricPlugin.Crashlytics.js
304 /plugins/cordova-plugin-android-permissions/www/permissions-dummy.js
304 /plugins/cordova-plugin-app-version/www/AppVersionPlugin.js
304 /plugins/cordova-plugin-camera/www/CameraConstants.js
304 /plugins/cordova-plugin-camera/www/CameraPopoverOptions.js
304 /plugins/cordova-plugin-camera/www/Camera.js
304 /plugins/cordova-plugin-camera/src/browser/CameraProxy.js
304 /plugins/cordova-plugin-device/www/device.js
304 /plugins/cordova-plugin-device/src/browser/DeviceProxy.js
304 /plugins/cordova-plugin-dialogs/www/notification.js
304 /plugins/cordova-plugin-dialogs/www/browser/notification.js
304 /plugins/cordova-plugin-file/www/DirectoryEntry.js
304 /plugins/cordova-plugin-file/www/DirectoryReader.js
304 /plugins/cordova-plugin-file/www/Entry.js
304 /plugins/cordova-plugin-file/www/File.js
304 /plugins/cordova-plugin-file/www/FileEntry.js
304 /plugins/cordova-plugin-file/www/FileError.js
304 /plugins/cordova-plugin-file/www/FileReader.js
304 /plugins/cordova-plugin-file/www/FileSystem.js
304 /plugins/cordova-plugin-file/www/FileUploadOptions.js
304 /plugins/cordova-plugin-file/www/FileUploadResult.js
304 /plugins/cordova-plugin-file/www/FileWriter.js
304 /plugins/cordova-plugin-file/www/Flags.js
304 /plugins/cordova-plugin-file/www/LocalFileSystem.js
304 /plugins/cordova-plugin-file/www/Metadata.js
304 /plugins/cordova-plugin-file/www/ProgressEvent.js
304 /plugins/cordova-plugin-file/www/fileSystems.js
304 /plugins/cordova-plugin-file/www/requestFileSystem.js
304 /plugins/cordova-plugin-file/www/resolveLocalFileSystemURI.js
304 /plugins/cordova-plugin-file/www/browser/isChrome.js
304 /plugins/cordova-plugin-file/www/browser/Preparing.js
304 /plugins/cordova-plugin-file/src/browser/FileProxy.js
304 /plugins/cordova-plugin-file/www/fileSystemPaths.js
304 /plugins/cordova-plugin-file/www/browser/FileSystem.js
304 /plugins/cordova-plugin-file-opener2/www/plugins.FileOpener2.js
304 /plugins/cordova-plugin-file-transfer/www/FileTransferError.js
304 /plugins/cordova-plugin-file-transfer/www/FileTransfer.js
304 /plugins/cordova-plugin-file-transfer/www/browser/FileTransfer.js
```

Only one window is opened with index page contents. When I close the window, only then the second one comes. And here is additional output after that:

```
SIM: APP_HOST disconnected from the server
SIM: Creating sim-host.js: Existing file found and is up-to-date.
200 /simulator/index.html (gzip)
304 /simulator/plugin/cordova-plugin-geolocation/OpenLayers.js
304 /simulator/sim-host.js
304 /simulator/thirdparty/webcomponents.min.js
304 /simulator/thirdparty/socket.io.js
304 /simulator/plugin/cordova-plugin-geolocation/sim-host.css
200 /simulator/sim-host.css (gzip)
200 /simulator/sim-host-theme.css (gzip)
304 /simulator/plugin/cordova-plugin-geolocation/compass.png
SIM: SIM_HOST connected to the server
SIM: Emitting 'init' to SIM_HOST
SIM: Setup handlers for SIM_HOST
SIM: Handling pending emits for SIM_HOST
SIM: Setup handlers for APP_HOST
SIM: Subscribing to a disconnected APP_HOST wanting 'exec'
SIM: Subscribing to a disconnected APP_HOST wanting 'plugin-message'
SIM: Subscribing to a disconnected APP_HOST wanting 'plugin-method'
SIM: Starting live reload.
SIM: Emitting 'resize-viewport' to DEBUG_HOST (pending connection)
SIM: Emitting 'plugin-message' to APP_HOST (pending connection)
SIM: Emitting 'plugin-message' to APP_HOST (pending connection)
SIM: SIM_HOST disconnected from the server
```

My environment:
Ubuntu 16.04 LTS
Node 8.9.1
cordova-simulate 0.3.13

**Comment by ezze** [comment]
The code is working if I open second window `http://localhost:8000/simulator/index.html` manually in a separate tab otherwise I get `undefined` errors in browser's console.

**Comment by luigi37** [comment]
+1
If I close the window then the second appear...

**Comment by ruslan-bikkinin** [comment]
Hi guys. This issue was fixed in cordova-serve@2.0.0, but we are using ^1.0.1 for now. New version includes several breaking changes so we need to change our API calls as well and make a proper testing after bumping version. We have added this in our roadmap and we will come back to it soon.

**Comment by iTOYS** [comment]
Fixed in cordova-simulate@0.4.0
